### ISSUE TITLE
[BUG] problematic load more

### ISSUE THREAD

**Issue by yoshimo** [TwidereProject/TwidereX-Android#293]
due to load limits sometimes there are gaps in the Timeline.
Then a button, load more appears.
The problem is, if you press it, unread tweets scroll out of view.
Would be nice if the view was locked and the refresh automatic once we are past the rate limit again.

**Comment by guguxfr** [comment]
I agree, same feeling here. At least, reloading more than 20 tweets can be easily done I guess, only a variable to modify ? Maybe 100 tweets at a time should be a good value ?
